src/main/java/org/opensearch/flowframework/rest/RestCreateWorkflowAction.java

```diff
@@ -242,7 +244,9 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
                     XContentBuilder exceptionBuilder = ex.toXContent(channel.newErrorBuilder(), ToXContent.EMPTY_PARAMS);
                     channel.sendResponse(new BytesRestResponse(ex.getRestStatus(), exceptionBuilder));
                 } catch (IOException e) {
-                    String errorMessage = "IOException: Failed to send back create workflow exception";
+                    String errorMessage = ParameterizedMessageFactory.INSTANCE.newMessage(
+                        "IOException: Failed to send back create workflow exception"
```

Comment:
No need to rewrite this one (and many similar), EXCEPT... if you want, you could create a constant in the `CommonValue` class, something like `FAILED_TO_SEND_EXCEPTION` with the value "IOException: Failed to send back {} exception".  Then here you'd add the string "create workflow" and in the other similar ones you'd change it (get, delete, etc.).